modules/ROOT/pages/getting-started.adoc

```diff
@@ -49,7 +47,7 @@ Because this project is a Maven project, it contains a `pom.xml` file that conta
 <parent>
   <groupId>org.mule.extensions</groupId>
   <artifactId>mule-modules-parent</artifactId>
-  <version>1.6.3</version>
+  <version>1.9.0</version>
```

Comment:
this section needs to explain (or refer to) that for connectors to be used in 4.4 or 4.6 parent poms 1.4.x or 1.6.x need to be used. Need to keep that until 4.6 goes EOL